scripts/myscript.js

```diff
@@ -0,0 +1 @@
+console.log("Hello World, this Emmy Leke with HNGi7 ID 05176 using Javascript for stage 2 task");
```

Comment:
Please your HNG ID  should be in the form `HNG-02576`

Reply:
I have made the fixes.